options/js/options.js

```diff
@@ -1,63 +1,64 @@
-/* eslint-disable brace-style */
-/* eslint-disable no-plusplus */
-/* eslint-disable no-console */
-/* eslint-disable vars-on-top */
-/* eslint-disable eqeqeq */
-var recentSearchQueries = [];
-var recentSearchQueryUrls = [];
-var toggle = document.getElementById("theme");
-
-if (!localStorage.getItem("theme")) { localStorage.setItem("theme", "light"); }
-
-if (localStorage.getItem("theme") == "light") {
+var recentSearchQueries=[];
+var recentSearchQueryUrls=[];
+var toggle = document.getElementById('theme');
+
+if(!localStorage.getItem('theme'))
+    localStorage.setItem('theme', 'light');
+
+if(localStorage.getItem('theme') == 'light')
+{
     console.log("light");
     toggle.checked = false;
-} else {
+}
+else
+{
     console.log("dark");
     toggle.checked = true;
 }
 
-// eslint-disable-next-line no-unused-vars
 function handleThemeChange(event) {
-    if (toggle.checked == false) {
-        localStorage.setItem("theme", "light");
-    }
-    else { localStorage.setItem("theme", "dark"); }
+    if (toggle.checked==false)
+        localStorage.setItem('theme', 'light');
+    else
+        localStorage.setItem('theme', 'dark');
 }
 
-// eslint-disable-next-line func-names
 document.addEventListener("DOMContentLoaded", function () {
-    // eslint-disable-next-line no-shadow
-    var toggle = document.getElementById("theme");
-    toggle.addEventListener("click", handleThemeChange);
+    var toggle = document.getElementById('theme');
+    toggle.addEventListener('click', handleThemeChange);
 });
 
 var historyListElement = document.getElementById("historyList");
 var clearHistory = document.querySelector("#clearButton");
 
-recentSearchQueries = (JSON.parse(localStorage.getItem("search")));
-recentSearchQueryUrls = (JSON.parse(localStorage.getItem("link")));
+recentSearchQueries=(JSON.parse(localStorage.getItem('search')));
+recentSearchQueryUrls=(JSON.parse(localStorage.getItem('link')));
 
-historyListElement.textContent = "";
-var count = 0;
-// eslint-disable-next-line func-names
-recentSearchQueries.forEach(function (entr) {
-    var aTag = document.createElement("a");
-    aTag.setAttribute("target", "_blank");
-    aTag.setAttribute("href", recentSearchQueryUrls[count]);
+historyListElement.textContent="";
+var count=0;
+recentSearchQueries.forEach(function(entr)
```

Comment:
so the whole with making the array name as plural was we can write element name inside our functional loops as singular. 
Ex - `recentSearchQueries.forEach(function(query)..`
So each element of recentSearchQueries is a query.

Always keep this thing in mind it is never never never bad to write big variable or function names (ex - popular repos have it (like react-redux.)[https://github.com/reduxjs/react-redux/blob/master/src/components/connectAdvanced.js#L168] have function names which are so self explanatory). They just need to as clear as possible. Its not an issue you will learn to do this slowly as you go through more and more codebases. 

Also maybe (Airbnb style guide)[https://github.com/airbnb/javascript#the-javascript-style-guide-guide] has mentioned such tips in an awesome way. Ex - something as basic as making your boolean variable name start with `is` gives a lot of clarity to your code. 
```
const isValid = true;
if (isValid) {
 // do this
}
```